packages/preview/src/browser/preview-link-normalizer.ts

```diff
@@ -27,7 +28,12 @@ export class PreviewLinkNormalizer {
         try {
             if (!this.urlScheme.test(link)) {
                 const location = documentUri.parent.resolve(link).path.toString();
-                return new MiniBrowserEndpoint().getRestUrl().resolve(location).toString();
+                if (environment.electron.is()) {
```

Comment:
I'm not a fan on runtime checks, should I rather use different module containers for this or is this fine?

Reply:
cc @AlexTugarev